modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -49,19 +48,21 @@ If you have external client providers such as PingFederate configured for your e
 
 *Best Practice*
 
-To secure your APIs by leveraging the multiple client providers feature, create one external client provider per environment. Assign the existing client provider to your production environment and add the new providers to other environments, such as sandbox, and QA. 
+To secure your APIs, create one external client provider per environment. Assign the existing client provider to your production environment and add the new providers to other environments, such as sandbox, and QA. However, check the possible impact on managed APIs and their consumers. 
 
-However, before changing any existing client providers, ensure that you check the impact on managed APIs and their consumers. Before you implement multiple client providers, see the <<guidelines,guidelines>>.
+Avoid using the same IdP in different environments. 
```

Comment:
This message is not completely accurate, the best practice is to use different IDPs for productive and non-productive environments. 
It is okay to associate the same IDP or with several productive environments or with several non productive ones. We need to avoid the association of a certain IDP with productive and non-productive envs.

Reply:
Hi @franciscoSoler, thanks! I think I fixed it. But please let me know if I'm still off.